.github/workflows/testing.yml

```diff
@@ -214,6 +214,22 @@ jobs:
           fetch-depth: 0
         if: matrix.tarantool != 'brew' && steps.cache.outputs.cache-hit != 'true'
 
+      - name: Patching tarantool 1.10.x for succesful build
+        run: |
+          # These steps fix the problem with tarantool build described in #6576
```

Comment:
Nit: I suggest to place a full link to the issue, because it is another repository.

Reply:
Placed.